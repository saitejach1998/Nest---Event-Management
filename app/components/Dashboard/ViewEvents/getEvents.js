

// Change YOUR_API_KEY_HERE to your apiKey
const url =
  "";

export async function getEvents(params) {
  if(params['type'] == 'global'){
    url = "http://192.168.43.209:3000/getglobalevents";
  }
  else if(params['type'] == 'local'){
    url="";
  }
  else if(params['type'] == 'self'){
    url=""
  }
  else{
    
  }

/*
  let result =  await fetch(url
         ,{
             method: 'POST',
         headers: {
           Accept: 'application/json',
           'Content-Type': 'application/json',
         },
         body: JSON.stringify(params)
          
       })
       
       .catch(err => console.warn(err));
      
 console.log(result)
  return result.articles;
  */
}
