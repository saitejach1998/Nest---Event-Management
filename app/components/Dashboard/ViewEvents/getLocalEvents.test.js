import React from 'react';
import getEvents  from './getLocalEvents.js';

import renderer from 'react-test-renderer';

describe('getEvents  component', function() {
    it('renders a SignupCompont ', function() {
      const tree = renderer.create(
        <getEvents />
        ).toJSON();
    expect(tree).toMatchSnapshot();
    });
  });