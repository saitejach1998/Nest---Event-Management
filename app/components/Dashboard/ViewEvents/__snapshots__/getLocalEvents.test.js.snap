// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getEvents  component renders a SignupCompont  1`] = `<getEvents />`;
